src/services/webdav/body_request_type.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[derive(Debug, Clone)]
+pub enum BodyRequestType {
```

Comment:
Not a good enum to me.

Reply:
Do you mean the naming? Or it is unnecessary?

Without this enum, we just use string for HTTP method?

Reply:
> Or it is unnecessary?

It's unnecessary to me.

Reply:
Sounds good. Removed!